src/Components/Categories.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Categories = () => (
-  <div>
-    <button type="button">Check Status</button>
-  </div>
-);
-
+const Categories = () => {
+  const categories = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <button type="button" onClick={() => dispatch(checkStatus())}>
+        Check Status
+      </button>
+      <h2>{categories}</h2>
+    </div>
+  );
+};
```

Comment:
- [ ] Kindly check this error on the categories page. Update and remove the error 
![image](https://user-images.githubusercontent.com/62705947/216630440-a754dc91-c349-4534-b95f-840ecf0c0ab1.png)

Reply:
- [ ] The logo is broken on your readme file, kindly update it 
![image](https://user-images.githubusercontent.com/62705947/216630627-41781138-23ca-4434-bf37-aed538bb43df.png)

Reply:
> * [ ]  Kindly check this error on the categories page. Update and remove the error
>   ![image](https://user-images.githubusercontent.com/62705947/216630440-a754dc91-c349-4534-b95f-840ecf0c0ab1.png)

Hi @GzimAsani Thank you very much for your feedback. I have fixed the errors on the categories page

Reply:
> * [ ]  The logo is broken on your readme file, kindly update it
>   ![image](https://user-images.githubusercontent.com/62705947/216630627-41781138-23ca-4434-bf37-aed538bb43df.png)

Hey @GzimAsani Thank you for the observation. I have fixed the logo